File: SDKs/github.com/api.github.com/1.1.4/go/pkg/models/operations/actionsgetworkflowrun.go
package operations

import (
	"openapi/pkg/models/shared"
)

type ActionsGetWorkflowRunPathParams struct {
	Owner string `pathParam:"style=simple,explode=false,name=owner"`
	Repo  string `pathParam:"style=simple,explode=false,name=repo"`
	RunID int64  `pathParam:"style=simple,explode=false,name=run_id"`
}

type ActionsGetWorkflowRunRequest struct {
	PathParams ActionsGetWorkflowRunPathParams
}

type ActionsGetWorkflowRunResponse struct {
	ContentType string
	StatusCode  int64
	WorkflowRun *shared.WorkflowRun
}

File: SDKs/github.com/ghes-3.1/1.1.4/typescript/src/sdk/models/operations/actionsgetworkflowrun.ts
import { Metadata, SpeakeasyBase } from "../../../internal/utils/utils";
import * as shared from "../shared";


export class ActionsGetWorkflowRunPathParams extends SpeakeasyBase {
  @Metadata({ data: "pathParam, style=simple;explode=false;name=owner" })
  owner: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=repo" })
  repo: string;

  @Metadata({ data: "pathParam, style=simple;explode=false;name=run_id" })
  runId: number;
}


export class ActionsGetWorkflowRunRequest extends SpeakeasyBase {
  @Metadata()
  pathParams: ActionsGetWorkflowRunPathParams;
}


export class ActionsGetWorkflowRunResponse extends SpeakeasyBase {
  @Metadata()
  contentType: string;

  @Metadata()
  statusCode: number;

  @Metadata()
  workflowRun?: shared.WorkflowRun;
}
